apps/services/search-indexer/src/environments/environment.prod.ts

```diff
@@ -8,4 +8,5 @@ export const environment: Environment = {
   dictRepo: 'https://api.github.com/repos/island-is/elasticsearch-dictionaries',
   s3Folder: '',
   locales: ['is', 'en'],
+  configPath: './config',
```

Comment:
Is there a different config file for prod?

(Since local dev references the grandparent directory)

Reply:
Yes when developing it uses the default project structure but when the project has been built then it looks like this
```
config/
migrate.js
main.js
```